src/chapters/exceptions/introduction.rst

```diff
@@ -0,0 +1,161 @@
+.. index:: ! exception, ! throw
+
+Exceptions
+==========
+
+Like most programming languages, C# includes exceptions and exception handling tooling. Some exception objects
+are provided by .NET itself, while others we may encounter from external C# libraries, or write ourselves.
```

Comment:
This could be broken down to something like: "Some exception objects are provided by .NET itself. Other exception objects we may get from external C# libraries. We may even write these objects ourselves."